frontend-public/src/components/common/DocumentTable.js

```diff
@@ -1,25 +1,28 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
-import { Table } from "antd";
+import { Table, Icon, Popconfirm, Button } from "antd";
 import CustomPropTypes from "@/customPropTypes";
 import { formatDate } from "@/utils/helpers";
 import downloadFileFromDocumentManager from "@/utils/actionlessNetworkCalls";
-import * as Strings from "@/constants/strings";
+import * as String from "@/constants/strings";
```

Comment:
What's the reason for switching this to `String`? I'd argue that `Strings` is preferable, as we shouldn't be modifying JS constructors imo

Reply:
oops, meant to do the opposite for the exact reason. my bad.